packages/reports/src/Components/Form/CompositeCalendar/side-list.tsx

```diff
@@ -1,16 +1,28 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { toMoment } from '@deriv/shared';
-import ListItem from './list-item.jsx';
+import ListItem from './list-item';
 
-const isActive = (from, to, flag) => {
+type TItem = {
+    value: string;
+    label: string;
+    onClick: () => void;
+    duration: number;
+};
+
+type TSideList = {
+    from: number;
+    items: Array<TItem>;
+    to: number;
+};
+
+const isActive = (from: number, to: number, flag: number) => {
     if (flag === 0) {
         return toMoment().endOf('day').unix() === to && from === null;
     }
     return Math.ceil(to / 86400) - Math.ceil(from / 86400) === flag;
 };
 
-const SideList = ({ items, from, to }) => (
+const SideList = ({ items, from, to }: TSideList) => (
     <ul className='composite-calendar__prepopulated-list'>
         {items.map(item => {
             const { duration, ...rest_of_props } = item;
```

Comment:
just for better code readability . 
```suggestion
            const { duration, label, onClick } = item;
```
and then change two lines after like this: 
`return <ListItem key={duration} is_active={is_active} label={label} onClick={onClick} />;`